src/ReaderScreen/utils/gutkaScript.js

```diff
@@ -25,13 +25,15 @@ window.addEventListener(
   false
 );
 
-(function scrollToPosition() {
-  setTimeout(function () {
-    let scrollY = (document.body.scrollHeight - window.innerHeight) * ${position};
-    window.scrollTo(0, scrollY);
-    curPosition = scrollY;
-  }, 50);
-})();
+window.onload = () => {
+  (function scrollToPosition() {
+    setTimeout(function () {
+      let scrollY = (document.body.scrollHeight - window.innerHeight) * ${position};
+      window.scrollTo(0, scrollY);
+      let curPosition = scrollY;
+    }, 50);
+  })(); 
+}
```

Comment:
Why do we need a setTimeout here for 50ms here? 

Also, this can be converted to arrow function syntax. And since its a named function, I would suggest, defining the function first, and doing the function call in separate line for better code readability.
```javascript
window.onload = () => {
   const scrollToPosition = () => {
       ....
    }
    scrollToPosition();
}
```